src/bbs-plus/proof.ts

```diff
@@ -1,35 +1,35 @@
 /**
  * Proof of knowledge of signature protocol
  */
-import { SignatureG1 } from './signature';
-import { SignatureParamsG1 } from './params';
+import { BBSPlusSignatureG1 } from './signature';
+import { BBSPlusSignatureParamsG1 } from './params';
 import {
-  bbsInitializeProofOfKnowledgeOfSignature,
-  bbsGenProofOfKnowledgeOfSignature,
-  bbsChallengeContributionFromProtocol,
-  bbsChallengeContributionFromProof,
-  bbsVerifyProofOfKnowledgeOfSignature,
-  BbsPoKSigProtocol,
+  bbsPlusInitializeProofOfKnowledgeOfSignature,
+  bbsPlusGenProofOfKnowledgeOfSignature,
+  bbsPlusChallengeContributionFromProtocol,
+  bbsPlusChallengeContributionFromProof,
+  bbsPlusVerifyProofOfKnowledgeOfSignature,
+  BbsPlusPoKSigProtocol,
   VerifyResult
 } from '@docknetwork/crypto-wasm';
 import { BBSPlusPublicKeyG2 } from './keys';
 import { BytearrayWrapper } from '../bytearray-wrapper';
 
-export class PoKSigProtocol {
-  value: BbsPoKSigProtocol;
+export class BBSPlusPoKSignatureProtocol {
+  value: BbsPlusPoKSigProtocol;
 
-  constructor(protocol: BbsPoKSigProtocol) {
+  constructor(protocol: BbsPlusPoKSigProtocol) {
     this.value = protocol;
   }
 
   static initialize(
     messages: Uint8Array[],
-    signature: SignatureG1,
-    params: SignatureParamsG1,
+    signature: BBSPlusSignatureG1,
+    params: BBSPlusSignatureParamsG1,
     encodeMessages: boolean,
     blindings?: Map<number, Uint8Array>,
```

Comment:
Can you put default values for both `blindings` and `revealed` like you have in PS? Same for BBS.